src/components/grid/gridRowContent.js

```diff
@@ -205,20 +185,30 @@ export const GridRowContent = props => {
           className='ef-session-name-link ef-session-name-mobile'
           onPress={onToggle}
         />
+
         <InfoRow onPress={onToggle}>
           <SessionLocationSmall
             className='ef-session-location-mobile'
             session={session}
             textClass='ef-session-location'
             iconGap={5}
           />
-          { hasExtraContent && (
-            <ToggleIcon
-              className='ef-session-expand-toggle'
-              Element={Chevron}
-              height={23}
-              width={18}
-            />
+          { isOpen ? (
+            <EvfButton
+              title='none'
+              onClick={onToggle}
+              buttonNameForGa={'Close Session Details - ' + session.name}
```

Comment:
Can't we remove most of this?
I thought we were just going to have a buttonType "sessionDetailsChevronOpen" and "sessionDetailsChevronCloseed"
and then do the logic in Eventsforce. No need for buttonNameForGa and sessionDetailsChevronIsOpen props in the session component.